File: Gladiator_Expenses.py
def calculate_gladiator_expenses(lost_fights, helmet_price, sword_price, shield_price, armor_price):
    total_helmet_broken = lost_fights // 2
    total_sword_broken = lost_fights // 3
    total_shield_broken = lost_fights // (2 * 3)
    total_armor_broken = total_shield_broken // 2
    expenses = (
        total_helmet_broken * helmet_price
        + total_sword_broken * sword_price
        + total_shield_broken * shield_price
        + total_armor_broken * armor_price
    )
    return expenses

# Read input values
number_of_lost_fights = int(input())
helmet_price = float(input())
sword_price = float(input())
shield_price = float(input())
armor_price = float(input())

# Calculate expenses using the function
expenses = calculate_gladiator_expenses(number_of_lost_fights, helmet_price, sword_price, shield_price, armor_price)

# Print the total expenses
print(f"Gladiator expenses: {expenses:.2f} aureus")

File: GladiatorExpenses.cpp
#include <iostream>
#include <iomanip>  // Include the header for setprecision

double calculateGladiatorExpenses(int lostFights, double helmetPrice, double swordPrice, double shieldPrice, double armorPrice) {
    int totalHelmetBroken = lostFights / 2;
    int totalSwordBroken = lostFights / 3;
    int totalShieldBroken = lostFights / (2 * 3);
    int totalArmorBroken = totalShieldBroken / 2;
    double expenses = totalHelmetBroken * helmetPrice +
                      totalSwordBroken * swordPrice +
                      totalShieldBroken * shieldPrice +
                      totalArmorBroken * armorPrice;
    return expenses;
}

int main() {
    int numberOfLostFights;
    double helmetPrice, swordPrice, shieldPrice, armorPrice;

    // Read input values
    std::cin >> numberOfLostFights >> helmetPrice >> swordPrice >> shieldPrice >> armorPrice;

    // Calculate expenses using the function
    double expenses = calculateGladiatorExpenses(numberOfLostFights, helmetPrice, swordPrice, shieldPrice, armorPrice);

    // Print the total expenses with 2 decimal places
    std::cout << "Gladiator expenses: " << std::fixed << std::setprecision(2) << expenses << " aureus" << std::endl;

    return 0;
}
